tpm/tpm.go

```diff
@@ -76,9 +76,22 @@ func GetKeys(rw io.ReadWriter) ([]Handle, error) {
 	return unpackKeyHandleList(b)
 }
 
+// PcrExtend extends a value into the right PCR by index.
+func PcrExtend(rw io.ReadWriter, i uint32, p pcrValue) ([]byte, error) {
+	in := []interface{}{i, p}
+	var d pcrValue
+	out := []interface{}{&d}
+	_, err := submitTPMRequest(rw, tagRQUCommand, ordExtend, in, out)
```

Comment:
It's a little more idiomatic in Go to merge this line with the if statement in the following line:

if _, err := submitTPMRequest(...); err != nil {
...
}

Looking back at the file, I see there are places where I didn't do this original and should have, but I'd like to not add more of those places. :)